index.html

```diff
@@ -230,13 +230,14 @@ <h3>Conceptual Framework</h3>
 
         <p>
 This section outlines the core concept utilized by the status list
-mechanism described in this document. At the most basic level, status
+mechanism described in this specification. At the most basic level, status
 information for all <a>verifiable credentials</a> issued by an <a>issuer</a>
-are expressed as simple binary values. The <a>issuer</a> keeps a bitstring
-list of all <a>verifiable credentials</a> it has issued. Each
-<a>verifiable credential</a> is associated with a position in the list. If
-the binary value of the position in the list is 1 (one), the
-<a>verifiable credential</a> is revoked, if it is 0 (zero) it is not revoked.
+are expressed as items in a list. The <a>issuer</a> manages a list
```

Comment:
```suggestion
is expressed as items in a list. The <a>issuer</a> manages a list
```

Reply:
This (`are` -> `is`) is part of my https://github.com/w3c/vc-bitstring-status-list/pull/106#discussion_r1437920379

Reply:
Applied.